2-mandatory/1-create-functions.js

```diff
@@ -44,7 +57,19 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(numbers) {
+  const formattedNumbers = [];
+
+  for (const number of numbers) {
```

Comment:
Try again, using `map()`.
Generally, when you need to convert an array to another array with the same number of elements, you should use `map`